pints/_mcmc/_adaptive_covariance.py

```diff
@@ -154,67 +136,52 @@ def tell(self, fx):
             # Clear proposal
             self._proposed = None
 
+            # Set r to zero
+            self._r = float('-Inf')
```

Comment:
r needs to be defined in the constructor too

Reply:
Yep, done.